src/witan/models/dem/ccm/inputs.clj

```diff
@@ -44,6 +44,12 @@
    :witan/key :future-mortality-trend-assumption
    :witan/schema s/NationalTrendsSchema})
 
+(definput in-future-fert-trend-1-0-0
```

Comment:
I can't see a CSV in the default_data sets that seems to match this. Am I just blind?